src/Relation/Nullary.agda

```diff
@@ -10,7 +10,10 @@
 
 module Relation.Nullary where
 
-open import Agda.Builtin.Equality
+open import Agda.Builtin.Equality using (_≡_)
+open import Agda.Builtin.Maybe public
+  using (nothing; just)
```

Comment:
That seems like a good idea.

Reply:
Thanks for the endorsement, @JacquesCarette .
I'm personally comfortable with multiple renamed versions of the same underlying structure, distinguished as to mode-of-use, but I recognise that that perhaps conflicts with existing usage eg in the haskell- or agda- ecosystems, so I wasn't about to make a 'controversial' change for its own sake without some discussion/agreement (esp. on names!)

Reply:
Unfortunately, I think we shouldn't be renaming them and re-exporting them from such a central module otherwise we're going to run into exactly the same issue we had with renaming `_,_` in `Data.Nat`, i.e. autocomplete will start filling in `undefined` rather than `nothing` when the user is not expecting it.

Reply:
Sigh... of course. But the type renaming is (nevertheless?) OK?

Reply:
Oh, good spot, I had not noticed that 'public' ! Indeed, that would cause more confusion. I was expecting the rename to be local to this file when I agreed it was a good idea.